### PR TITLE
Implemented a logic app which was previously just in v2

### PR DESCRIPTION
**Pull request by raffertyuy**
## Purpose
Implemented 2 logic apps
1. Upload --> transcode --> video indexer
2. Publish and Get Streaming URL

These were previously in https://github.com/Azure-Samples/media-services-dotnet-functions-integration, but not in the v3 repo.

There are also a lot bug fixes to the advanced-vod-functions when I was implementing this.

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
1. Deploy the azure functions
2. Deploy the logic apps and configure the connectors
3. Upload an MP4 file in a storage watch container.
4. Run the logic app trigger

## What to Check
Verify that the following are valid
* Check if a new multi-bitrate asset is created
* Check if a new indexed asset appears in your VI account
* Check if a new streaming locator is created (using the 2nd logic app)

## Other Information
High level explanation of the logic app here: https://raffertyuy.com/raztype/creating-an-azure-media-services-upload-workflow-using-azure-storage-and-logic-apps/

### CODE REVIEW

**Comment by raffertyuy** [comment]
> [CLA assistant check](https://cla.opensource.microsoft.com/Azure-Samples/media-services-v3-dotnet-core-functions-integration?pullRequest=12)
> Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.microsoft.com/Azure-Samples/media-services-v3-dotnet-core-functions-integration?pullRequest=12) before we can accept your contribution.
> 
> ❌ raffertyuy [sign now](https://cla.opensource.microsoft.com/Azure-Samples/media-services-v3-dotnet-core-functions-integration?pullRequest=12)
> You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.microsoft.com/check/Azure-Samples/media-services-v3-dotnet-core-functions-integration?pullRequest=12) it.

Can't sign at the moment, the URL is down

**Comment by raffertyuy** [comment]
CLA is up, just signed.

File: logic-app-using-workflow-functions/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+services: media-services,logic-app
+platforms: dotnetcore
+author: raffertyuy
+---
+
+# Media Services: Integrating Azure Media Services with Azure Functions & Azure Logic Apps
+
+This project contains Logic Apps that coverts-to-media-asset, encodes and publishes media files that you upload in an Azure Storage.
```
Comment (johndeu) [inline logic-app-using-workflow-functions/README.md:9]: converts not coverts

File: logic-app-using-workflow-functions/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+The project includes Logic Apps ARM templates that is dependent on the [advanced-vod-workflow](../advanced-vod-workflow) functions. There are 2 projects, one for uploading and encoding and another for publishing.
+
+This Media Services Functions example is based on AMS REST API v3 on Azure Functions v2.
```
Comment (johndeu) [inline logic-app-using-workflow-functions/README.md:13]: Is the project still up to date? Do we need to refresh this with latest SDK versions?

File: logic-app-using-workflow-functions/README.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+---
+services: media-services,logic-app
+platforms: dotnetcore
+author: raffertyuy
+---
+
+# Media Services: Integrating Azure Media Services with Azure Functions & Azure Logic Apps
+
```
Comment (johndeu) [inline logic-app-using-workflow-functions/README.md:8]: Could use a lot more description details here about the Use Case scenario that is being demonstrated with this sample

File: logic-app-using-workflow-functions/README.md (hunk 2 above)
Comment (xpouyat) [reply logic-app-using-workflow-functions/README.md:13]: I am updating code to Azure functions v3, and .Net Core 3.1, with latest SDKs

File: logic-app-using-workflow-functions/README.md (hunk 1 above)
Comment (xpouyat) [reply logic-app-using-workflow-functions/README.md:9]: Fixed

**Pull request merged**